applications/StructuralMechanicsApplication/custom_utilities/constitutive_law_utilities.h

```diff
@@ -21,6 +21,8 @@
 #include "includes/ublas_interface.h"
 #include "includes/node.h"
 #include "geometries/geometry.h"
+#include "includes/constitutive_law.h"
+#include "structural_mechanics_application_variables.h"
```

Comment:
this is not used in this file

Reply:
ping, this should still be removed (you did it in the other PR)

Reply:
still here :)

Reply:
the one with variables is necessary for compiling. The unused is the const_law

Reply:
then you need to include it in the claw that actually uses those variables
this file (`constitutive_law_utilities.h`) does NOT use them so it should not include this file
this is about the reduction of dependencies

Reply:
+1 for Philipp's comment.  this should really not be here